suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -85,18 +86,15 @@ export const AccountImportLoadingScreen = ({
             if (!ignore) {
                 if (fetchedAccountInfo?.success) {
                     if (networkSymbol === 'eth') {
-                        fetchedAccountInfo.payload.tokens?.forEach(token => {
+                        fetchedAccountInfo.payload.tokens?.forEach(token =>
                             dispatch(
-                                updateFiatRatesThunk({
-                                    ticker: {
-                                        symbol: 'eth',
-                                        tokenAddress: token.contract as TokenAddress,
-                                    },
-                                    rateType: 'current',
-                                    localCurrency: fiatCurrency,
+                                getTokenDefinitionThunk({
+                                    networkSymbol: 'eth',
```

Comment:
also `matic`?

Reply:
Matic is blacklisted in the app so it would be dead code?

Reply:
Wont be forever. If not inclusing this, I suggest to add this to matic issue to resilve later

Reply:
same comment as above applies.